ansible/roles/os_zabbix/vars/template_os_linux.yml

```diff
@@ -337,3 +337,8 @@ g_template_os_linux:
     url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_yum_update.asciidoc"
     priority: info
     description: "One or more security updates need to be applied to the host"
+
+  - name: "pcp items send failed on {HOST.NAME}"
+    expression: "{Template OS Linux:kernel.uname.sysname.nodata(15m)}=1"
+    url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_pcp_ping.asciidoc"
+    priority: average
```

Comment:
Ok, I spent a lot of time last night thinking about this.

For posterity, here are the tradeoffs:
Good:
* this is reusing an item we're already sending

Bad:
* nodata is going to cause this trigger to fire for all cases where the heartbeat trigger will fire (doubling them up)
* nodata is more expensive than max


Given the above tradeoffs, I think I prefer keeping it the way it was, so I suggest getting rid of these changes.